kie-pmml-trusty/kie-pmml-kie-dependencies/pom.xml

```
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kie-pmml-trusty</artifactId>
+    <groupId>org.kie</groupId>
+    <version>7.48.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kie-pmml-kie-dependencies</artifactId>
```

What about rename it 
```suggestion
  <artifactId>kie-pmml-kie-internal</artifactId>
```

I think `kie-dependencies` is misleading because for example `kie-pmml-evaluator-api` already depends on `kie-api` (same for `kie-pmml-api`) so this is not the only module with `kie` relations